contracts/test/v0.8/automation/AutomationRegistry2_2.test.ts

```diff
@@ -170,7 +171,7 @@ let chainModuleBase: ChainModuleBase
 let arbitrumModule: ArbitrumModule
 let optimismModule: OptimismModule
 let streamsLookupUpkeep: StreamsLookupUpkeep
-let automationUtils: AutomationUtils
+let automationConv: AutomationCompatibleUtils
```

Comment:
the var doesn't match the contract anymore here

Reply:
done